src/Home.js

```diff
@@ -28,39 +34,30 @@ import FaucetDripForm from './components/FaucetDripForm';
 // import FabricIdentityManager from './components/FabricIdentityManager';
 // import PortalMenu from './components/PortalMenu';
 
-class Home extends FabricComponent {
-  constructor (props) {
-    super(props);
-
-    this.settings = Object.assign({
+export default function Home (props) {
```

Comment:
Would love to preserve use of classes if possible

Reply:
Updated via 5eaeacfb63ad952aa0ccd5b5e1793ae1fd184c7b && cfcc31e3e6439e8e0d99ca371cba8d05e2dacc8a